ang/api4Explorer/Explorer.js

```diff
@@ -876,9 +878,8 @@
         newLine = "\n" + _.repeat(' ', indent),
         code = '\\' + info.class + '::' + action + '(',
         args = _.cloneDeep(info.class_args || []);
-      if (params.checkPermissions === false) {
-        args.push(false);
-      }
+        // Always shows implicit true permissions check for PHP
+        args.push((params.checkPermissions === false) ? false : true);
```

Comment:
This indentation is wrong, needs to be un-indented by 2 spaces.

Reply:
Fixed